src/lib/bridge/Bridge.svelte

```diff
@@ -74,6 +75,7 @@
     $: deposit = L2 === false ? true : false;
 
     const selectToken = () => {
+        amountToBridge = "0";
```

Comment:
You import the constant ZERO which is "0" so maybe use that one instead of "0"?

Reply:
Done for all occurrences!